doc/source/cluster/running-applications/job-submission/ray-client.rst

```diff
@@ -26,13 +26,23 @@ By changing ``ray.init()`` to ``ray.init("ray://<head_node_host>:<port>")``, you
    do_work.remote(2)
    #....
 
+When to use Ray Client
+----------------------
+
+Ray Client should be used when you want to connect a script or an interactive shell session to a **remote** cluster.
```

Comment:
Should we just say interactively develop on a remote cluster? Specifically you shouldn't use it to "fire and forget" a script on a remote cluster (you should just submit it as a job instead)

Reply:
Removed mentions of running scripts with client